opflexagent/utils/bridge_managers/ovs_manager.py

```diff
@@ -31,12 +32,28 @@ class OvsManager(bridge_manager_base.BridgeManagerBase,
     def __init__(self):
         super(OvsManager, self).__init__()
 
-    def initialize(self, host, ovs_config, opflex_conf):
+    def initialize(self, host, opflex_conf, agent_state):
```

Comment:
It seems weird to pass some configuration as an argument (opflex_conf), but then use the global confguration reference to try and get the OVS configuration. It might be cleaner to pass a less-specific configuration? In other words, instead of having an opflex_conf argument, just have "conf", or "configuration", and then reference configuration.OVS and configuration.OPFLEX.

Reply:
I fixed this as you suggested.